deploy/single-node/config.cue

```diff
@@ -101,6 +106,9 @@ package k8s
 
 	// List of router interfaces to resolve to names
 	interfaceMap: [...#InterfaceMap]
+
+	ingressType: *"goflow" | "portmirror"
```

Comment:
I think we want to support setups that run both - how about leaving goflow and conditionally deploying portmirror if its config is present?

Reply:
Yeah sounds better